frontend/components/Editor.js

```diff
@@ -318,7 +319,7 @@ export class Editor extends Component {
                         state.selected_cells = []
 
                         for (let cell of new_cells) {
-                            state.cell_inputs_local[cell.cell_id] = cell
+                            state.notebook.cell_inputs[cell.cell_id] = cell
```

Comment:
Shouldn't this be in an `update_notebook()` call as well, for it to be sent to the other clients?